tests/unit/common/browserInfoHelper.test.js

```diff
@@ -208,27 +219,30 @@ describe("BrowserInfoHelper", () => {
       setCandidateString(
         "abc xyz 123 567 789 777 127.0.0.1 randomstring somestring"
       );
-      webRtcConnectionStub.mockReturnValue(MockRTCPeerConnection);
+      global.RTCPeerConnection = MockRTCPeerConnection;
       expect(await getDeviceLocalIPAsString()).toEqual("789");
     });
   });
 
   it("getMTDTaxReturnWithParameter", async () => {
-    jest.spyOn(globalsUtil, "getNavigator").mockReturnValue({
+    navigatorSpy.mockImplementation(() => ({
       plugins: getMockBrowserPluginDetails(),
       doNotTrack: "yes",
-    });
-    jest.spyOn(globalsUtil, "getWebRTCConnection").mockReturnValue(MockRTCPeerConnection);
-    jest.spyOn(globalsUtil, "getWindow").mockReturnValue({
+    }));
+
+    windowSpy.mockImplementation(() => ({
       devicePixelRatio: 2,
       innerWidth: 1009,
-      innerHeight: 1013,
-    });
-    jest.spyOn(globalsUtil, "getScreen").mockReturnValue({
+      innerHeight: 1013
+    }));
+
+    global.RTCPeerConnection = MockRTCPeerConnection;
+    
+    screenSpy.mockImplementation(() => ({
       width: 1019,
       height: 1021,
-      colorDepth: 17,
-    });
+      colorDepth: 17
+    }))
```

Comment:
Minor: Please add the semi colon at the end to follow the same pattern everywhere.